tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -30,6 +30,21 @@ def create_test_datafile(filename, stt_user, stt, section='Active Case Data'):
 
 def generate_parser_error(datafile, line_number, schema, error_category, error_message, record=None, field=None):
     """Create and return a ParserError using args."""
+    if type(field) == list:
+        fields_json = {
+            "friendly_name": {
+                getattr(f, 'name', None): getattr(f, 'friendly_name', None) for f in field
+            }
+        }
+    elif field is not None:
```

Comment:
Is this needed for if `field` is a string or something non-iterable? Could we make the loop on L36 work without a type check and simplify this whole block?